com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PollExecuteAppStatusExecution.java

```diff
@@ -160,7 +160,42 @@ private void stopApplicationIfSpecified(ExecutionWrapper execution, CloudControl
             messageType = MessageType.STDOUT;
             text = attr;
         }
-        return new Pair<>(messageType, text);
+        return new Marker(messageType, text);
     }
 
+    private static class AppExecution {
```

Comment:
This is a wrapper for a Status and a status message.
From my point of view, the status message, should be a property of the status, there's no need to wrap them in a separate class.